### PR TITLE
[WIP] Plant portal markdown for workshop

### PR DESCRIPTION
**Pull request by mtekman**
Based on singlecell and rna portal

### CODE REVIEW

File: index-plants.md [hunk 1]
```diff
@@ -5,10 +5,62 @@ subdomain: plants
...
+# Welcome to Galaxy for Plant Biology
+{:.no_toc}
+
+The Plant Anaylsis Workbench is a comprehensive set of analysis tools and consolidated workflows. The workbench is based on the [Galaxy framework](https://galaxyproject.org){:target="_blank"},
+which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.
+
+The current version comprises more than XX  bioinformatics tools dedicated to different research areas of single cell biology.
+
+This service is a joint project between different groups from the [XXXXX](http://www.test.ac.uk){:target="_blank"}, the [XXXXX](http://www.test.ac.uk){:target="_blank"}, [XXXXX](http://www.test.ac.uk){:target="_blank"}, and the [University of Freiburg](https://galaxyproject.eu/freiburg/){:target="_blank"}.
+The server is part if the European Galaxy server and is maintained by the [RNA Bioinformatics Center (RBC)](https://www.denbi.de/network/rna-bioinformatics-center-rbc){:target="_blank"} as part of [de.NBI](https://www.denbi.de){:target="_blank"} and [ELIXIR](http://elixir-europe.org){:target="_blank"}.
```
Comment (beatrizserrano) [inline index-plants.md:19]: @bgruening can you check this?

Comment (bgruening) [reply index-plants.md:19]: The following should be mentioned:

* dataplant 
* BioDaten 
* VIB? need to ask Frederik
* IPK? need to ask Uwe
* Timo?

Comment (mtekman) [reply index-plants.md]: I added the above partners, but could not find Timo...?

**Comment by mtekman** [comment]
I added as many materials as I could find, all that is missing is two rows from the table:

* mRNA differential expression analysis
* miRNA differential expression and target prediction analysis

everything else is linked, except for those which are waiting on upstream changes:

* A GTN video link for the Plates, Batches, and Barcodes slides
* A youtube video link for the Plant tutorial Hands-on

File: index-plants.md (hunk 1 above)
Comment (bgruening) [reply index-plants.md]: https://www.bio.uni-kl.de/organisation/mitarbeiter/m-o/muehlhaus-timo/

**Comment by bgruening** [comment]
Thanks @mtekman. I will push the button, but please add new stuff as they become available.

**Comment by beatrizserrano** [comment]
we need to add the other tutorials, 2 from the workshop and a previous one by @gallardoalba

**Pull request merged**